### PR TITLE
Add account to variables

### PR DESCRIPTION
**Pull request by hweawer**

### CODE REVIEW

File: src/variables.py [hunk 1]
```diff
@@ -107,6 +107,7 @@
     'HEALTHCHECK_SERVER_PORT': HEALTHCHECK_SERVER_PORT,
     'MAX_CYCLE_LIFETIME_IN_SECONDS': MAX_CYCLE_LIFETIME_IN_SECONDS,
     'DEPOSIT_MODULES_WHITELIST': DEPOSIT_MODULES_WHITELIST,
+    'ACCOUNT': ACCOUNT,
```
Comment (F4ever) [inline src/variables.py:110]: better
```suggestion
    'ACCOUNT': ACCOUNT.address,
```

Comment (hweawer) [reply src/variables.py]: Done

File: src/variables.py [hunk 2]
```diff
@@ -107,6 +107,7 @@
     'HEALTHCHECK_SERVER_PORT': HEALTHCHECK_SERVER_PORT,
     'MAX_CYCLE_LIFETIME_IN_SECONDS': MAX_CYCLE_LIFETIME_IN_SECONDS,
     'DEPOSIT_MODULES_WHITELIST': DEPOSIT_MODULES_WHITELIST,
+    'ACCOUNT': '' if ACCOUNT is None else ACCOUNT.address(),
```
Comment (F4ever) [inline src/variables.py:110]:
```suggestion
    'ACCOUNT': '' if ACCOUNT is None else ACCOUNT.address,
```

Comment (hweawer) [reply src/variables.py]: Done

**Review by F4ever — APPROVED** [review]

**Pull request merged**